src/shared/helpers.jsx

```diff
@@ -0,0 +1,47 @@
+import { ButtonsAdmin } from '../components/Buttons/ButtonsAdmin/ButtonsAdmin';
+import { ButtonsStatusUpdate } from '../components/Buttons/ButtonsStatusUpdate/ButtonsStatusUpdate';
+
+export async function filterMembers(items) {
+  const users = await items;
+
+  return users.map((item) => ({
+    ...item,
+    name: `${item.name} ${item.lastName}`,
+    actions: <ButtonsAdmin id={item.id} />,
+  }));
+}
+
+export async function filterProgress(items) {
+  const traks = await items;
```

Comment:
why do you await variable ? you already should have resolved result. fix it here and below please

Reply:
It's not needed here either. Removed async await. Thanks.